test/fixup-svg-string.js

```diff
@@ -27,3 +27,20 @@ test('fixupSvgString should make parsing fixtures not throw', t => {
     });
     t.end();
 });
+
+test('fixupSvgString should correct invalid mime type', t => {
+    const filePath = path.resolve(__dirname, './fixtures/invalid-cloud.svg');
+    const svgString = fs.readFileSync(filePath)
+        .toString();
+    const fixed = fixupSvgString(svgString);
+    const dontFix = fixupSvgString('<text>data:img/png is not a mime type</text>');
```

Comment:
im not sure that i would put this in the same test -- i feel like this is a second test - its possible to pass one and fail the other and that's "okay" in the sense that a future editor could determine the error they made that broke the dont fix scenario while knowing they didn't break the general fix scenario